### DISCUSSION TITLE
Combo Box with empty string option is not working properly

### DISCUSSION THREAD

**Discussion by erzenhaxhi** [taiga-family/taiga-ui#6162] [Q&A] [answered]
Hi, when combo box has one empty string option that value is selected by default and if we select another option and we open the dropdown again the previous value is overridden by empty string.

Can we handle this? Or this need to be reported as bug?

[here is the example](https://stackblitz.com/edit/angular-9atujt)

**Comment by waterplea** [comment, marked as answer]
That's a bug, but can you clarify why you need such a case? You can provide `[strictMatcher]="null"` so it will never auto select a value as you type for now.
